srcpkgs/suricata/template

```diff
@@ -1,21 +1,38 @@
 # Template file for 'suricata'
 pkgname=suricata
 version=4.1.2
-revision=1
+revision=2
 build_style=gnu-configure
-configure_args="--disable-gccmarch-native"
-hostmakedepends="pkg-config"
-makedepends="libnet-devel libpcap-devel pcre-devel libyaml-devel libcap-ng-devel file-devel"
-short_desc="Next Generation Intrusion Detection and Prevention Engine"
+configure_args="--disable-gccmarch-native $(vopt_enable rust_support rust)
+ $(vopt_if luajit --enable-luajit) $(vopt_if lua --enable-lua)
+ $(vopt_if hiredis --enable-hiredis)"
+hostmakedepends="pkg-config $(vopt_if rust_support cargo)"
+makedepends="libnet-devel libpcap-devel pcre-devel libyaml-devel libcap-ng-devel
+ liblz4-devel liblzma-devel file-devel jansson-devel nss-devel $(vopt_if luajit LuaJIT-devel)
+ $(vopt_if lua lua-devel) $(vopt_if hiredis hiredis-devel) $(vopt_if hiredis libevent-devel)"
+short_desc="Suricata is a network IDS, IPS and NSM engine"
 maintainer="Andrea Brancaleoni <abc@pompel.me>"
-license="GPL-2"
+license="GPL-2.0-only"
 homepage="https://suricata-ids.org/"
 distfiles="https://www.openinfosecfoundation.org/download/$pkgname-$version.tar.gz"
 checksum=73575b041a50cc48a2a53f6503ab4d355166d7acbd4997cd04045f848f8bea96
+build_helper="$(vopt_if rust_support rust)"
```

Comment:
build_helper should be directly below  build_style=

Reply:
i'm quite sure `xlint` complained about `build_helper` not being below checksum... i move it upwards anyway